modin/conftest.py

```diff
@@ -487,11 +487,11 @@ def set_async_read_mode(request):
 
 
 @pytest.fixture
-def set_min_partition_size(request):
-    old_min_partition_size = MinPartitionSize.get()
-    MinPartitionSize.put(request.param)
+def set_min_row_partition_size(request):
```

Comment:
Have tests using this fixture relied on resizing along the row axis only?

Reply:
Yes, you can take a look at test_astype_categorical_issue5722 and test_cat_codes_issue5650.